BUILD.gn

```diff
@@ -8,16 +8,33 @@ challenge_bypass_build_path = "$root_gen_dir/challenge_bypass_ristretto/out"
 
 challenge_bypass_lib_path = challenge_bypass_build_path
 
+challenge_bypass_platform = ""
+
 if (is_win) {
   challenge_bypass_lib_path += "/x86_64-pc-windows-msvc"
+  challenge_bypass_platform = "Windows"
 } else if (is_mac) {
   challenge_bypass_lib_path += "/x86_64-apple-darwin"
+  challenge_bypass_platform = "macOS"
 } else if (is_linux) {
-  challenge_bypass_lib_path += "/x86_64-unknown-linux-musl"
+  challenge_bypass_lib_path += "/x86_64-unknown-linux-gnu"
+  challenge_bypass_platform = "Linux"
 } else if (is_android) {
-  challenge_bypass_lib_path += "/x86_64-linux-android"
+  if (target_cpu == "arm" || target_cpu == "arm64") {
+    challenge_bypass_lib_path += "/arm-linux-androideabi"
```

Comment:
is this right? Shouldn't one be arm and one be arm64? I guess @serg would be the best person to ask

Reply:
I did ask @serg, but I have now added arm, arm64, x86 and x86_64